hs_tools_resource/templates/pages/toolresource.html

```diff
@@ -7,180 +7,159 @@
-                <div>
-                    {% if issues_page_url %}
-                        <h4><strong>Issues Page URL:</strong></h4>
-                        <hr class="tool-metadata-hr">
-                        <div class="row">
-                            <div class="col-sm-8">{{ issues_page_url.value|urlize }}</div>
-                        </div>
-                        <hr style="border:0">
-                    {%  endif %}
-                </div>
-                <div>
-                    {% if mailing_list_url %}
-                        <h4><strong>Mailing List URL:</strong></h4>
-                        <hr class="tool-metadata-hr">
-                        <div class="row">
-                            <div class="col-sm-8">{{ mailing_list_url.value|urlize }}</div>
-                        </div>
-                        <hr style="border:0">
-                    {%  endif %}
-                </div>
-                <div>
-                    {% if roadmap %}
-                        <h4><strong>Roadmap:</strong></h4>
-                        <hr class="tool-metadata-hr">
-                        <div class="row">
-                            <div class="col-sm-8">{{ roadmap.value|urlize }}</div>
-                        </div>
-                        <hr style="border:0">
-                    {%  endif %}
-                </div>
-{#                <div>#}
-{#                    {% if show_on_open_with_list %}#}
-{#                        <h4><strong>Shown on Open With List:</strong></h4>#}
-{#                        <hr class="tool-metadata-hr">#}
-{#                        <div class="row">#}
-{#                            <div class="col-sm-8">{{ show_on_open_with_list.value|urlize }}</div>#}
-{#                        </div>#}
-{#                        <hr style="border:0">#}
-{#                    {%  endif %}#}
-{#                </div>#}
-            </div>
-        </div>
-    {%  endif %}
+        {% if homepage_url %}
+            <div class="col-lg-6 col-xs-12">
+                <h4>Home Page URL:</h4>
+                <div class="row">
+                    <div class="col-sm-8">{{ homepage_url.value|urlize }}</div>
+                </div>
+                <br>
+            </div>
+        {% endif %}
+        {% if url_base %}
+            <div class="col-lg-6 col-xs-12">
+                <h4>App-launching Resource Level URL Pattern:
+                    <a href='/terms#AppURLPattern' target='_blank'><font size='3'>Help</font></a>
+                </h4>
+                <div class="row">
+                    <div class="col-sm-8">{{ url_base.value }}</div>
+                </div>
+                <br>
+            </div>
+        {% endif %}
+        {% if url_base_aggregation %}
+            <div class="col-lg-6 col-xs-12">
+
+                <h4>App-launching Aggregation Level URL Pattern:
+                    <a href='/terms#AppURLPattern' target='_blank'>
+                        <font size='3'>Help</font></a>
```

Comment:
outsource this to CSS

Reply:
Agree. For now I am changing this to a view function so that the UI can make a separate call to get this limited set of resource level metadata needed for updating the UI. Later (not in this PR) I think it would be better to change this view function to return data only for a specified metadata element - so that the UI can access data for any supported metadata element.